### PR TITLE
Accessibility checks

### PR DESCRIPTION
**Pull request by Azonkeu**
This pull request was created by the following contributors: 
@oluwajuwon8
@abdessalem1998
@Azonkeu 

Accessibility checks have been done and here are the results:

- [ ] Page titles - no issues found

- [ ] Image text alternatives - https://github.com/Azonkeu/personal-portfolio/pull/9/commits/b36b383d7f3ed343f8185683123ee539a4b3b77d#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L253-R253

- [ ] Text headings – https://github.com/Azonkeu/personal-portfolio/pull/9/commits/b36b383d7f3ed343f8185683123ee539a4b3b77d#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L253-R253

- [ ] Contrast color – https://github.com/Azonkeu/personal-portfolio/pull/9/commits/b36b383d7f3ed343f8185683123ee539a4b3b77d#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR736-R1156

- [ ] Resize - no issues found

- [ ] Interaction – https://github.com/Azonkeu/personal-portfolio/pull/9/commits/9fef34a1362e4269b0f92afe4baac65b64c17e1e#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR1241-R2223

- [ ] Moving-content – https://github.com/Azonkeu/personal-portfolio/pull/9/commits/c652aa9a15038ccbe76ee835f1851554beb65840#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL1509-R1511

- [ ] Multimedia – no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Ornela Portfolio</title>
```
Comment (bimbolabuari) [inline index.html:7]: Why don't we make this more descriptive: 🤔

- Better page titles:
     "Ornela's Portfolio| full-stack web developer" 👍

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Azonkeu , 

You have done a good job so far 🚀  

## Highlights 🎉 

-  Correct Github flow ✔️ 
-  Linter checked ✔️ 
-  Made adjustments according to project requirements ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Comment by omar-labana** [comment]
Hi @Azonkeu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — COMMENTED** [review]
Hi @Azonkeu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has been approved already.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding! 👩‍💻 

keep trying

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**